src/index.html

```diff
@@ -1,13 +1,58 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="product_card-styles/price-block.css">
+    <link rel="stylesheet" href="product_card-styles/rating-block.css">
+    <link rel="stylesheet" href="product_card-styles/purchase-block.css">
+    <link rel="stylesheet" href="product_card-styles/image-block.css">
+    <link rel="stylesheet" href="product_card-styles/title-block.css">
+    <link rel="stylesheet" href="product_card-styles/product_code-block.css">
+    <link rel="stylesheet" href="product_card-styles/product_card_global.css">
+    <link rel="stylesheet" href="main.css">
```

Comment:
import all these links in the `main.css` instead of connecting them directly to html